examples/cfd/external_aerodynamics/domino/src/train.py

```diff
@@ -103,10 +103,12 @@ def loss_fn(
     num = torch.sum(mask * (output - target) ** 2.0, dims)
     if loss_type == "rmse":
         denom = torch.sum(mask * target**2.0, dims)
+        loss = torch.mean(torch.sqrt(num) / torch.sqrt(denom))
     else:
```

Comment:
```suggestion
    elif loss_type == "mse":
```

And add `else: raise ValueError("...")`

Reply:
Done